### DISCUSSION TITLE
Installation problem

### DISCUSSION THREAD

**Discussion by armornet** [ZsgsDesign/NOJ#518] [Q&A] [answered]
Hello, i have installed NOJ and all steps went fine but when i access the site on the homepage it is stuck at "Preparing NOJ" but i am able to access the admin panel.. i have not installed judger api or judge server.. are these judger api or judge server required for the home page to open..? please advise

**Comment by ZsgsDesign** [comment, marked as answer]
Normally it's dependencies but in that case, you would not be able to open the admin portal.
Providing more information would be helpful. You can press `F12` to open the dev panel to see all errors in the frontend, if you don't understand the errors you can post screenshots here.

You can post issues and we will reply to you much quicker, we didn't quite understand this new discussion feature for now.
